storage/rocksdb/ha_rocksdb.h

```diff
@@ -494,6 +494,12 @@ class ha_rocksdb : public my_core::handler {
     Pointer to the original TTL timestamp value (8 bytes) during UPDATE.
   */
   char m_ttl_bytes[ROCKSDB_SIZEOF_TTL_RECORD];
+  /*
+    The TTL timestamp value can change if the explicit TTL column is
+    updated. If we detect this when updating the PK, we indicate it here so
+    we know we must always update any SK's.
```

Comment:
This is an important perf impact and I think we need to remember to document this later on in a public wiki (in fact we should start writing a wiki for TTL).  This means MySQL is forced to update all SKs even if none of the actual SK fields have changed.